### ISSUE TITLE
Accidental 2 Linux home roots, how to reset the command prompt to the right one?

### ISSUE THREAD

**Issue by 98-f355-f1** [Microsoft/WSL#3382]
DELETED Original

**Comment by 98-f355-f1** [comment]
I can't find an answer to this anywhere:
I think that I have 2 different instances of Bash on my desktop system with the latest OS10 Spring update.
I installed Debian for WSL and it works great when I bring up the Debian bash.. and goes to the correct xxxx@xxxxx~$ (home root) for the installation of Debian.

BUT if I bring up PowerShell or CMD and type wsl or bash, it goes to a different WSL root.

I think that it's related to an earlier attempt to install Ubuntu, then I uninstalled it, and now have the Debian version.   Any help to get the PowerShell and CMD to go to the correct Debian path?  In other words, how to change the path that PS and CMD use when type in "wsl" or "bash"?

Otherwise, I have put the Debian on a windows laptop and it is working wonderfully... I am not able to program things using Linux POSIX, that I hadn't been able to use before... LOVE IT!!

MICROSOFT fan for life!
